### PR TITLE
testcase updation & get req

### PR DESCRIPTION
**Pull request by vkosuri**
https://github.com/bulkan/robotframework-requests/issues/90

### CODE REVIEW

File: src/RequestsLibrary/RequestsKeywords.py [hunk 1]
```diff
@@ -704,14 +704,17 @@ def _utf8_urlencode(self, data):
...
-                data = data
-        else:
-            data = data
 
         return data
+
+    def _is_json(data):
+        try:
+            json_obj = json.loads(data)
+        except ValueError, e:
```
Comment (bulkan) [inline src/RequestsLibrary/RequestsKeywords.py]: something about this looks not quite right...

**Comment by vkosuri** [comment]
I am skipping if data is in json format by using this proc or it is about exception?

File: src/RequestsLibrary/RequestsKeywords.py (hunk 1 above)
Comment (vkosuri) [inline src/RequestsLibrary/RequestsKeywords.py]: This fix about https://github.com/bulkan/robotframework-requests/issues/90
>problem for us as well. All of our "data=" lines are already formatted JSON. I was able to keep our current "data=" by changing the following in RequestsKeywords.py

**Comment by oleduc** [comment]
Maybe he's refering to the use of the comma notation for the exception. This will not work in python 3.4. 
Use "except ValueError as e"

But that shouldn't matter as rf doesn't support python 3.4 yet.

**Comment by bulkan** [comment]
I just find it weird that you have to first try to parse a string as json if it fails its not json.

**Pull request merged**